src/main/scala/org/camunda/feel/impl/builtin/StringBuiltinFunctions.scala

```diff
@@ -42,7 +43,8 @@ object StringBuiltinFunctions {
     "split"            -> List(splitFunction),
     "extract"          -> List(extractFunction),
     "trim"             -> List(trimFunction),
-    "uuid"             -> List(uuidFunction)
+    "uuid"             -> List(uuidFunction),
+    "encode base64"    -> List(encodeBase64Function)
```

Comment:
:wrench: We could rename the function to `to base64()` to align with other data transformation functions, like `to json()` ([ref](https://github.com/camunda/feel-scala/issues/825)). 

What do you think about changing the name?

Reply:
Yes as it was just a proposal and I wasnt happy with it either.